collaboration/communication.md

```diff
@@ -1,54 +1,92 @@
-        so share only what you are comfortable sharing online
-        you can share the rest in confidence with you group by another channel
--->
-
 # Communication
 
 ______________________________________________________________________
 
 ## Communication Schedule
 
-| Day | How | The topic of discussion | | --- | :-: | ----------------------- |
-| | | |
+| **Day**       | **How**              | **Topic of Discussion**            |  
+|----------------|----------------------|-------------------------------------|  
+| Friday/Sunday  | Video call (Zoom)    | Update on progress                 |  
+| Everyday       | Slack                | Quick questions, announcements     |  
 
-## Communication Channels
+______________________________________________________________________
 
-how often will we get in touch on each channel, and what we will discuss there:
+## Communication Channels
 
-- **Issues**:
-- **Pull Requests**:
-- **Slack/Discord**:
-- **Video Calls**:
+**How often will we get in touch on each channel, and what we will discuss there:**  
+
+- **Issues**:  
+  - **Frequency**: Daily or as new issues arise.  
+  - **Topics Discussed**:  
+    - Bugs and their resolution.  
+    - Feature requests or enhancements.  
+    - Documentation or knowledge gaps.  
+    - General project-related discussions requiring tracking.  
+
+- **Pull Requests**:  
+  - **Frequency**: As needed when code changes are ready for review.  
+  - **Topics Discussed**:  
+    - Code review and feedback.  
+    - Suggestions for improvement or refactoring.  
+    - Discussions on implementation details.  
+    - Final approvals and merging criteria.  
+
+- **Slack/Discord**:  
+  - **Frequency**: Continuously throughout the workday.  
+  - **Topics Discussed**:  
+    - Quick questions or clarifications.  
+    - Updates on progress or blockers.  
+    - Sharing resources, links, or relevant information.  
+    - Informal discussions or team bonding in designated channels.  
+
+- **Video Calls**:  
+  - **Frequency**: Weekly or bi-weekly, or as needed for critical discussions.  
+  - **Topics Discussed**:  
+    - Project planning and roadmap alignment.  
+    - Sprint retrospectives or reviews.  
+    - Deep-dive discussions on complex issues.  
+    - Conflict resolution and decision-making.  
 
 ______________________________________________________________________
 
 ## Availability
 
-### Availability for calling/messaging
+### Availability for Calling/Messaging
+
+| **Day**      | **Monday** | **Tuesday** | **Wednesday** | **Thursday** | **Friday** | **Saturday** | **Sunday** |  
+|--------------|:----------:|:-----------:|:-------------:|:------------:|:----------:|:------------:|:----------:|  
+| Derek        | 12:00-2:00 | 12:00-2:00  | 12:00-2:00    | 12:00-2:00   | 12:00-2:00 | 12:00-2:00   | 12:00-2:00 |  
+| Safaa        | Yes        | Yes         | Yes           | No           | No         | No           | No         |  
```

Comment:
I think you should replace the "yes", "no" with  "available", "unavailable" here